CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.1.0] (IN PROGRESS)
```

Comment:
Should it be bugfix release?

Reply:
We haven't discussed if  we need to do a bugfix release or not. I'll raise this question on our standup meeting today